openedx/adg/lms/applications/management/commands/update_is_prerequisite_courses_passed.py

```diff
@@ -0,0 +1,71 @@
+""" Management command to update is_prerequisite_courses_passed flag in ApplicationHub """
+import logging
+import sys
+
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+from lms.djangoapps.courseware.courses import get_course
+from lms.djangoapps.grades.api import CourseGradeFactory
+from openedx.adg.lms.applications.models import ApplicationHub
+from openedx.adg.lms.course_meta.models import CourseMeta
+from student.models import CourseEnrollment
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Updates is_prerequisite_courses_passed flag in ApplicationHub.
+    """
+    def handle(self, *args, **kwargs):  # pylint: disable=unused-argument
+        pre_req_course_metas = CourseMeta.pre_reqs.all()
+
+        if not pre_req_course_metas:
+            sys.exit('Exiting!!! No pre-req courses found. But there must be some pre-reqs. Please add in admin.')
+
+        users_need_to_be_updated = self.get_users_need_to_be_updated(pre_req_course_metas)
+        pre_req_courses = [get_course(course_meta.course.id) for course_meta in pre_req_course_metas]
```

Comment:
1. Can we rename `pre_req_courses` to `pre_req_courses_descriptor`, since it is course from mongo

2. I think there is no need to get a courses descriptor, you can getgrades like this:

`CourseGradeFactory().read(user, course_key=course_meta.course.id)`